backend/src/main/java/com/google/rolecall/services/PerformanceServices.java

```diff
@@ -202,7 +202,7 @@ private PerformanceSection addPerformanceCastMember(PerformanceSection performan
     User user = userService.getUser(memberInfo.userId());
 
     PerformanceCastMember member = PerformanceCastMember.newBuilder()
-        .setOrder(position.getOrder())
+        .setOrder(memberInfo.order())
```

Comment:
Is this the cast position? (1st cast, 2nd cast, etc.)

Reply:
No, this is the dancer # field when you have many dancers in the same position. I thought this field was unused because it was always 0 with the test data I had. (So I tried to reuse it for position order.) This change restores the original code before my position order change. (I now get the position order from the code change below.)

Reply:
I do not think this is feature asked by AA. I am fine with the update but maybe we can also let them know as a new feature in the system.

Reply:
It is actually a bug fix, fixing the bug I introduced when I overwrote a field that was needed for the casting logic. My first "fix" gave us ordered positions, but broke the casts. This fix makes the cast logic work. I don't think Alvin Ailey is aware of the bug, (it was only in the system for a week and it didn't show easily) so there isn't much need to talk about the change.